src/Client/app/ThemeSwitch.tsx

```
@@ -1,44 +1,40 @@
-    return (
-      <IoMoon 
-        onClick={() => setTheme('dark')} 
-        style={{ cursor: 'pointer' }} 
-      />
-    )
-  }
+  return (
+    <button
+      onClick={() => setTheme('dark')}
+      style={{ display: 'flex', alignItems: 'center', gap: '0.5rem', background: 'none', border: 'none', padding: 0, cursor: 'pointer' }}
+    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)